tests/Client/UpdaterTest.php

```diff
@@ -7,29 +7,45 @@
 
 class UpdaterTest extends TestCase
 {
-
-    public function testUpdateSuccessful()
+    /**
+     * @return Updater
+     */
+    protected function getSystemInTest()
     {
         $mirrors = [
-        'mirror1' => [
-        'url_prefix' => 'http://localhost:8001',
-        'metadata_path' => 'metadata',
-        'targets_path' => 'targets',
-        'confined_target_dirs' => [],
-        ],
+            'mirror1' => [
+                'url_prefix' => 'http://localhost:8001',
+                'metadata_path' => 'metadata',
+                'targets_path' => 'targets',
+                'confined_target_dirs' => [],
+            ],
         ];
         $updater = new Updater('repo1', $mirrors);
-        $fixtureTarget = 'testtarget.txt';
-        $targetStream = fopen('data://text/plain,' . 'Test File', 'r');
-        $this->assertTrue($updater->validateTarget($fixtureTarget, $targetStream));
+        return $updater;
+    }
+
+    public function testRefreshRepository()
+    {
+        $updater = $this->getSystemInTest();
+        $this->assertTrue($updater->refresh());
     }
 
   /**
    * Tests that an error is thrown on an updated root.
    *
-   * @todo Remove this test when functionality is added.
+   * @todo Remove this test when updating root functionality is added.
    */
-    public function testUpdatedRootError()
+    public function testErrorOnUpdatedRoot()
+    {
+    }
+
+    /*
+    public function testValidateTarget()
     {
+        $updater = $this->getSystemInTest();
+        $fixture_target = 'testtarget.txt';
+        $target_stream = fopen('data://text/plain,' . 'Test File', 'r');
+        $this->assertTrue($updater->validateTarget($fixture_target, $target_stream));
     }
+    */
```

Comment:
Just noticed that this is commented out. Why?

Reply:
The newly emptied function it tests does not return true and failing tests is a bad look.

Reply:
(Coverage is maintained by the new `testRefreshRepository()`)

Reply:
Can we file an issue for it then and add a `@todo` comment that links said issue? Commented code is dead code.

Reply:
Can we just delete it for now? We won't forget to implement validating targets if we don't file an issue :wink:

Reply:
That's true; it's kind of critical to the functionality. 😛  I'm fine with just deleting it for now. I just feel very strongly about not having commented code or `@todo`s without documented issues for them. 🙂

Reply:
>  or @todos without documented issues for them.

Good to know

Reply:
```suggestion
```